### PR TITLE
Add Vantage Integration

### PR DESCRIPTION
**Pull request by brookemckim**
### What does this PR do?

Adds the Vantage Integration.

### Motivation

The Vantage integration will be GA later this month.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Working with Ana Wishnoff @ Datadog.

### CODE REVIEW

File: vantage/manifest.json
Comment (anawishnoff) [inline vantage/manifest.json:15]: For the images, would you be able to place them on a background or take new screenshots so that they have a 16:9 aspect ratio? This will ensure that they don't get cut off in the media carousel. More guidelines on media can be found [here](https://docs.datadoghq.com/developers/marketplace/#media-carousel-requirements).

**Review by anawishnoff — CHANGES_REQUESTED** [review]
A few review comments from my end. Thanks!

File: vantage/README.md
Comment (anawishnoff) [inline vantage/README.md:17]:
```suggestion
## Support
```

We only support a couple of specific h2 headers - support makes more sense here

File: vantage/README.md
Comment (anawishnoff) [inline vantage/README.md:13]:
```suggestion
### Configuration
```

File: vantage/manifest.json
Comment (anawishnoff) [inline vantage/manifest.json:11]:
```suggestion
    "description": "Import your Datadog costs and track them alongside other infrastructure spend",
```

Just a suggestion, but this language is more standard with other integration descriptions.

File: vantage/manifest.json
Comment (anawishnoff) [inline vantage/manifest.json:34]:
```suggestion
      "Offering::Integration",
      "Category::Cost Management",
      "Category::Cloud"
      
```
Suggesting a few categories to help with discoverability

**Review by anawishnoff — COMMENTED** [review]
A few more comments from the product side, then it looks good to go once docs and engineering are able to approve!

File: vantage/metadata.csv
Comment (bgoldberg122) [inline vantage/metadata.csv:1]: Hi team, thanks for opening up this PR! You can delete this file since you have nothing defined in `assets` in `manifest.json`

File: vantage/manifest.json
Comment (bgoldberg122) [inline vantage/manifest.json:45]: Could you please download the eula file, upload it to this PR, and then link to that version of the document?

**Review by bgoldberg122 — CHANGES_REQUESTED** [review]

File: vantage/manifest.json
Comment (bgoldberg122) [inline vantage/manifest.json:3]:
```suggestion
  "app_uuid": "2784a986-2673-4189-a3b8-320755f6c2b4",
```
This `app_uuid` has to match the `published_app_uuid` field of your private version of the app in the developer platform.

**Comment by bgoldberg122** [comment]
Regarding the github actions validation failure (`vantage/CHANGELOG.md does not exist.`), you must create a `CHANGELOG.md` file. Please see [this integration](https://github.com/DataDog/integrations-extras/blob/master/embrace_mobile/CHANGELOG.md) as an example.

**Review by bgoldberg122 — APPROVED** [review]
Looks good! Thanks for making the changes!

**Comment by anawishnoff** [comment]
Hey @brookemckim, apologies for not catching this earlier but for integrations using OAuth, it would be helpful to have an uninstallation section. I'd recommend adding the following to the end of the Overview section in the readme file:

```
### Uninstallation
<any Vantage-specific uninstallation steps that need to be taken>
To ensure this integration is fully uninstalled, be sure to revoke authorization in the Configure tab of this tile or in [Organization Settings > OAuth Apps](https://app.datadoghq.com/organization-settings/oauth-applications). 

Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys management page](https://app.datadoghq.com/organization-settings/api-keys).

```

We're working on making these steps automatic as a part of the integration uninstallation, but until then it's important we add this extra bit of context in.

File: vantage/CHANGELOG.md
Comment (anawishnoff) [inline vantage/CHANGELOG.md:1]:
```suggestion
# CHANGELOG - Vantage
```

File: vantage/CHANGELOG.md
Comment (anawishnoff) [inline vantage/CHANGELOG.md:3]:
```suggestion
## 1.0.0
```

File: vantage/CHANGELOG.md
Comment (anawishnoff) [inline vantage/CHANGELOG.md:5]:
```suggestion
```

File: vantage/CHANGELOG.md
Comment (anawishnoff) [inline vantage/CHANGELOG.md:7]:
```suggestion
* [FEATURE] adds Vantage integration.
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Adding suggestions to formatt the changelog file

**Review by anawishnoff — APPROVED** [review]

File: vantage/README.md
Comment (fanny-jiang) [inline vantage/README.md:7]: What do you think about moving the `Uninstallation` section closer to the bottom? After `Configuration` and before `Support`. I think the doc will flow better with the `Installation` section up top.

Comment (anawishnoff) [reply vantage/README.md:7]: I'm good with this suggestion - having this show up under the Configure tab makes more sense

File: vantage/README.md
Comment (buraizu) [inline vantage/README.md:9]:
```suggestion
To remove the Datadog integration from Vantage, visit the [integrations page][1] and click **remove**.
```

File: vantage/README.md
Comment (buraizu) [inline vantage/README.md:23]:
```suggestion
Once integrated, begin exploring your Datadog costs within Vantage. You can create filters for specific Datadog organizations and services alongside costs from any of the other supported Vantage providers.
```

File: vantage/manifest.json
Comment (buraizu) [inline vantage/manifest.json:11]:
```suggestion
    "description": "Import your Datadog costs and track them alongside other infrastructure spending",
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, made a few suggestions, mostly based on our [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md).

File: vantage/CHANGELOG.md
Comment (anawishnoff) [inline vantage/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 10-03-2022
```
Missed adding a date in my previous review, sorry about that!

File: vantage/README.md
Comment (buraizu) [reply vantage/README.md:5]: Just following up on this -- although `spend` as a noun is becoming more commonly used, it's not universally recognized in all dictionaries, such as Merriam-Webster for example:
https://www.merriam-webster.com/dictionary/spend

As such, this suggestion is focused around accessibility, as some users may not be familiar with this usage yet.

File: vantage/README.md
Comment (buraizu) [reply vantage/README.md:7]: From a documentation perspective, I agree that it makes more sense to relocate the `Uninstallation` section as mentioned here.

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for implementing those changes. I added some additional context behind the reasoning for the open suggestions, but please let me know if you have any questions.

**Review by buraizu — APPROVED** [review]

File: vantage/README.md
Comment (anawishnoff) [inline vantage/README.md:11]:
```suggestion
Visit [Vantage][4] to sign up for free. Once registered, visit the [Vantage integrations page][1] and add a Datadog integration. This guides you through the Datadog OAUTH2 flow to grant Vantage access to your billing and usage data.
```

File: vantage/README.md
Comment (anawishnoff) [inline vantage/README.md:19]:
```suggestion
To remove the Datadog integration from Vantage, visit the [Vantage integrations page][1] and click **remove**. Additionally, you'll want to uninstall this integration from Datadog by clicking the **Uninstall Integration** button below.
```

**Review by yzhan289 — APPROVED** [review]
LGTM! Should this be merged now? Looks like it has a `do-not-merge` label although the CHANGELOG mentions this is supposed to be released today?

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Two tiny final suggested changes after previewing your tile. Thanks!

**Review by anawishnoff — APPROVED** [review]

File: vantage/CHANGELOG.md
Comment (anawishnoff) [inline vantage/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 10-12-2022
```

**Pull request merged**
